fixtures/fun-generation/led-pot-12x1w-qcl-rgb-ww-40.json

```diff
@@ -0,0 +1,507 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Fun Generation LED Pot 12x1W QCL RGB WW 40°",
```

Comment:
The manual is for both "15°" and "40°", so it looks like they have identical DMX mappings.  Should we write one file for both, and include a note that the angle can be different?  Or include separate "physical" sections for each one?

Reply:
The differences are:

- Angles
- Article numbers
- Product URLs

I think one file is the better option because it avoids code duplication.

Reply:
I agree with consolidation, where possible, but I'm not sure exactly how it would work.  OFL does allow a "physical" section in the "modes" list, which seems like it would allow duplication of only the mode lists (channel names) -- that's still not ideal.

Is there any way in OFL to say "This entire DMX mapping is identical between these two fixtures, and they differ only in physical characteristics"?

Reply:
> Is there any way in OFL to say "This entire DMX mapping is identical between these two fixtures, and they differ only in physical characteristics"?

Maybe @FloEdelmann can answer this?

Reply:
Unfortunately not. However note that would also add maintainability issues if it later turned out to behave slightly differently in the DMX channels as well.

Reply:
Ok, so lets add the 15° version as a separate fixture once this get's merged 👍